src/deltacode/__init__.py

```diff
@@ -337,22 +416,26 @@ def to_dict(self):
         else:
             old_file = None
 
-        return OrderedDict([
-            ('status', self.status),
-            ('factors', self.factors),
-            ('score', self.score),
-            ('new', new_file),
-            ('old', old_file),
-        ])
+        return OrderedDict(
+            [
+                ("status", self.status),
+                ("factors", self.factors),
+                ("score", self.score),
+                ("new", self.file_to_dict(deltacode, self.new_file, new_file=True)),
+                ("old", self.file_to_dict(deltacode, self.old_file, new_file=False)),
+            ]
+        )
+
 
 class Stat(object):
     """
     Contains all the stats for the file changes in the new directory
     with respect to the old directory.
     """
+
     def __init__(self, new_files_count, old_files_count):
-        self.new_files_count = new_files_count
-        self.old_files_count = old_files_count
+        self.new_files_count = new_files_count[0]
```

Comment:
```suggestion
        self.new_files_count = new_files_count
```
The constructor should only accept ints, as explained above where `Stat()` is called.

Reply:
Okay changing it